### ISSUE TITLE
Curses crashes in skinny terminal, only on some posts

### ISSUE THREAD

**Issue by LeoTindall** [ihabunek/toot#27]
With a terminal of 75 columns or fewer (94 or up is OK, couldn't test more granularly due to time constraints but I suspect 80 is the magic number), some posts cause the Curses interface to emit the following message and crash when trying to display their contents.

```
Traceback (most recent call last):
  File "/usr/local/bin/toot", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.5/dist-packages/toot/console.py", line 285, in main
    run_command(app, user, command_name, args)
  File "/usr/local/lib/python3.5/dist-packages/toot/console.py", line 262, in run_command
    return fn(app, user, parsed_args)
  File "/usr/local/lib/python3.5/dist-packages/toot/commands.py", line 176, in curses
    TimelineApp(generator).run()
  File "/usr/local/lib/python3.5/dist-packages/toot/app.py", line 40, in run
    curses.wrapper(self._wrapped_run)
  File "/usr/lib/python3.5/curses/__init__.py", line 94, in wrapper
    return func(stdscr, *args, **kwds)
  File "/usr/local/lib/python3.5/dist-packages/toot/app.py", line 59, in _wrapped_run
    self.loop()
  File "/usr/local/lib/python3.5/dist-packages/toot/app.py", line 74, in loop
    self.select_next()
  File "/usr/local/lib/python3.5/dist-packages/toot/app.py", line 99, in select_next
    self.redraw_after_selection_change(old_index, new_index)
  File "/usr/local/lib/python3.5/dist-packages/toot/app.py", line 108, in redraw_after_selection_change
    self.draw_status_details(self.right, new_status)
  File "/usr/local/lib/python3.5/dist-packages/toot/app.py", line 210, in draw_status_details
    window.addstr(y, 1, '-' * (text_width + 2))
_curses.error: addwstr() returned ERR
```

I can't find anything that is consistently the same about these posts. Some have Emoji, some do not. Some have media, some do not. Some are from users on my instance and some from other instances. Some are boosts, some are original.

**Comment by ihabunek** [comment]
Thanks for reporting this. The curses app is woefully incomplete. See #28.

**Comment by ihabunek** [comment]
@LeoTindall This issue, as well as #25 and #26 should be fixed by PR #39, if you have some time check out the code and let me know if it works.

**Comment by ihabunek** [comment]
@LeoTindall this should be fixed in 0.17.0, please upgrade and test. I set the lower limit to 60 columns, which seems reasonable, to allow some room for the toots to be rendered nicely.

**Comment by LeoTindall** [comment]
Sorry for not responding before; yes this does seem to be fixed for my use case. Thanks so much!
